kiss

```diff
@@ -791,25 +802,23 @@ pkg_checksums() {
 pkg_verify() {
     # Verify all package checksums. This is achieved by generating a new set
     # of checksums and then comparing those with the old set.
-    verify_cmd="
-    NR==FNR{a[\$1];next}/^git  .*|^SKIP$/{next}!((\$1)in a){exit 1}
-    "
```

Comment:
Small unrelated change (`^git` should no longer appear in any checksums files). This was an old transitional change.

Reply:
The problem here is that many packages still haven't updated since the last checksums change.

Reply:
This should be fixable with a little script run over the repositories. (ie, delete all lines in checksums files that == `^git`). I didn't realize we still had packages using the old format.

Reply:
It's been almost a year since the code I removed was added (for reference): https://github.com/kisslinux/kiss/commit/d6917d5fe1558d0690a7dd001996d34540ed79d9